### PR TITLE
WIP Boot riemann

### PR DESCRIPTION
**Pull request by donbonifacio**
Still not working.

### CODE REVIEW

File: src/clanhr/analytics/metrics.clj [hunk 1]
```diff
@@ -72,11 +75,23 @@
...
+  "True if the lib should send stuff to riemann"
+  []
+  (env :clanhr-analytics-riemann))
+
 (defn- register
   "Registers a metric to librato"
   [env-name source event-name value description]
   (when (log-stdout?)
     (println (str "[" source "] " event-name " " value " - " description)))
+  (when (log-riemann?)
```
Comment (lamuria) [inline src/clanhr/analytics/metrics.clj:88]: maybe we should separate this in multiple functions

**Comment by lamuria** [comment]
:ok:

File: src/clanhr/analytics/metrics.clj (hunk 1 above)
Comment (donbonifacio) [inline src/clanhr/analytics/metrics.clj:88]: Good ideia @lamuria .
I do need to make this work first.

**Comment by donbonifacio** [comment]
Gonna close this. Maybe later.

**Pull request closed without merging**